### ISSUE TITLE
How to access the data from the "Associations" directly?

### ISSUE THREAD

**Issue by alatif-alatif** [fbdesignpro/sweetviz#163]
Hi,

thanks for creating and maintaining this project!

I have more than 100 variables and have difficulties to read the picture that gets generated in the "Associations" part of the html output.

Is there any possibility to access the data from within an internal structure of sweetviz?

E.g. like a DataFrame etc. that I could access before sweetviz generates the html output?

Thanks in advance!

**Comment by fbdesignpro** [comment]
You're right, this got should be an easy win as we already have the data! You should be able to access it through the `report._associations` variable, which is a map. 

I'll look into making this a bit better, but for now this should be giving you the data you need.
